node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -1204,15 +1207,13 @@ mod tests {
         running_test();
         let _guard = EnvironmentGuard::new();
         let _clap_guard = ClapGuard::new();
-        let home_dir = ensure_node_home_directory_exists(
+        let node_home_dir = ensure_node_home_directory_exists(
```

Comment:
I'd prefer `home_dir`. Also, let's change the function name to also say, `ensure_home_directory_exists`.